rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java

```diff
@@ -133,15 +132,9 @@ private CachingRlsLbClient(Builder builder) {
     backoffProvider = builder.backoffProvider;
     ChildLoadBalancerHelperProvider childLbHelperProvider =
         new ChildLoadBalancerHelperProvider(helper, new SubchannelStateManagerImpl(), rlsPicker);
-    if (rlsConfig.getRequestProcessingStrategy()
-        == RequestProcessingStrategy.SYNC_LOOKUP_CLIENT_SEES_ERROR) {
-      refCountedChildPolicyWrapperFactory =
-          new RefCountedChildPolicyWrapperFactory(
-              childLbHelperProvider, new BackoffRefreshListener());
-    } else {
-      refCountedChildPolicyWrapperFactory =
-          new RefCountedChildPolicyWrapperFactory(childLbHelperProvider, null);
-    }
+    refCountedChildPolicyWrapperFactory =
+        new RefCountedChildPolicyWrapperFactory(
+            childLbHelperProvider, new BackoffRefreshListener());
```

Comment:
The 2nd constructor arg no longer `@Nullable`?

Reply:
done.